notebooks/simone_pipeline.md

```diff
@@ -0,0 +1,134 @@
+# Summary of Simone's pipeline
+
+## Overview of methods needed
+
+1. 3d gaussian filter (`skimage.filters.gaussian` or `scipy.ndimage.gaussian`) -> `starfish.filters.gaussian` (already present)
+2. `ndimage.gaussian_laplace` -> `starfish.filters.gaussian_laplace`
+	1. Can we assume that any trimming of the image to fit within the dynamic range should be associated with the function that produces the aberration? E.g. `gaussian_laplace` can produce negative values -- should we wrap that function so that it doesn't have this characteristic?
+  2. If no, we will need some clean-up functions
+3. `max_projection` over z -> `starfish.image._stack` (currently present)
+4. Threshold finding. Is there an equivalent library function for the gradient calculation + trimming that this func does? -> `starfish.stats.calculate_peak_threshold_abs`. Replace `calculate` with your favorite word.
```

Comment:
sorry -- i don't understand the question. starfish.stats.calculate_peak_threshold_abs is not a real function yet right? what is it supposed to do?

Reply:
ah now i understand the question. maybe this lives as an optional thresholding method for the peak local max spot detector, instead of in starfish.stats. thoughts?

Reply:
Yea, that might make sense. We could make it a private method in the same module.